m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2070,10 +2088,13 @@ PROCEDURE BUILDSLTRANSFERLIST
          ELSE
           TRANREADONLYSTATUS(TRANCOUNT)="true"
 
-
          TRANACCOUNTNAME(TRANCOUNT)=SHARE:NICKNAME
          IF SHARE:NICKNAME="" THEN
           TRANACCOUNTNAME(TRANCOUNT)=SHARE:DESCRIPTION
+         CHARCHECK=TRANACCOUNTNAME(TRANCOUNT)
```

Comment:
Instead of doing the checking/scrubbing here, can we just do the check where we are outputting the json?

Reply:
@rrobison44 : Removed